ios-showcase-template/root/RootInteractor.swift

```diff
@@ -19,6 +19,18 @@ protocol RootInteractor: MenuListener {
     var router: RootRouter? { get set }
 }
 
+enum DocsURL: String {
+    case IdmDoc = "https://docs.aerogear.org/aerogear/latest/keycloak/index.html"
```

Comment:
Can we make the values here camelCase, I think that's the convention since Swift 3